cfg/cis-1.5/master.yaml

```diff
@@ -307,8 +321,15 @@ groups:
 
       - id: 1.1.21
         text: "Ensure that the Kubernetes PKI key file permissions are set to 600 (Scored)"
-        audit: "stat -c %n\ %a /etc/kubernetes/pki/*.key"
-        type: "manual"
+        audit: "stat -c permissions=%a /etc/kubernetes/pki/*.key"
```

Comment:
ditto, better use find, beside the audit command the use of multiple values and bitmask is great :)